sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyClient.java

```diff
@@ -740,9 +748,12 @@ public Key restoreKey(byte[] backup) {
      * <p>Lists the deleted keys in the key vault and for each deleted key prints out its recovery id.</p>
      * {@codesnippet com.azure.keyvault.keys.keyclient.listDeletedKeys}
      *
-     * @return A {@link List} containing all of the {@link DeletedKey deleted keys} in the vault.
+     * <p><strong>Samples to iterate DeletedKey by each page</strong></p>
```

Comment:
The sample description needs more description as to what its doing like the one above it, +  its doing it per page.